recipes/b2/portable/conanfile.py

```diff
@@ -53,49 +56,89 @@ def validate(self):
             raise ConanInvalidConfiguration(
                 "Option toolset 'cxx' and 'cross-cxx' requires 'use_cxx_env=True'")
 
+    def layout(self):
+        conan.tools.layout.basic_layout(self, src_folder="source")
+
     def source(self):
-        tools.get(**self.conan_data["sources"][self.version],
-                  strip_root=True, destination="source")
+        conan.tools.files.get(
+            self, **self.conan_data["sources"][self.version], strip_root=True)
```

Comment:
We include `destination=self.source_folder` in CCI recipes when using `get`.

Reply:
The default path is ".", not source_folder (virtually are same thing). Please, keep it explicit, because is easier to read. Also, it's what is adopted by ConanCenterIndex: https://github.com/conan-io/conan-center-index/blob/master/docs/package_templates/cmake_package/all/conanfile.py

Reply:
@uilianries same as https://github.com/conan-io/conan-center-index/pull/13758#discussion_r1006871164, let's update `package_templates` to match migration guide to avoid double work